cfg/config.go

```diff
@@ -3,11 +3,98 @@
 // that will be retrieved from  .env or ./cfg/config.yml
 package cfg
 
+import (
+	"bufio"
+	"log"
+	"os"
+	"strings"
+	"time"
+)
+
+// Load configs from a env file & sets them in environment variables .
+func LoadEnvVar() error {
+
+	f, err := os.Open(".env")
+
+	if err != nil {
+		log.Printf("%s", err)
+		return err
+	}
+
+	defer func() {
+		err := f.Close()
+		if err != nil {
+			log.Printf("%s", err)
+		}
+	}()
+
+	lines := make([]string, 0)
+	scanner := bufio.NewScanner(f)
+
+	for scanner.Scan() {
+		lines = append(lines, scanner.Text())
+	}
+
+	if err := scanner.Err(); err != nil {
+		log.Printf("%s", err)
+		return err
+	}
+
+	for _, l := range lines {
+		pair := strings.Split(l, "=")
+		os.Setenv(pair[0], pair[1])
+	}
+	return nil
+}
+
+var (
+	serverHost = os.Getenv("SERVHOST")
+	serverPort = os.Getenv("SERVPORT")
+	DBName     = os.Getenv("DBNAME")
+	DBUser     = os.Getenv("DDBUSER")
+	DBPassword = os.Getenv("DBPASSWORD")
+	DBPort     = os.Getenv("DBPORT")
+	DBHost     = os.Getenv("DBHOST")
+)
+
+// Server configuration description
+type Server struct {
```

Comment:
It seems like this struct completely coping http.Server struct, but without implemented method and some other fields. Why don't we just use http.Server struct instead?

Reply:
What if we name it `HTTP` @lifehou5e ? This module is responsible only for config rather than http.Server stuff with hanlders, etc.

Reply:
But renaming won't fix the issue. Why don't we just not use this struct at all then if it's not appropriate in this particular layer. We can use http.Server wherever we want it to through our code, so i am not sure if we need Server struct in cfg file at all.

Reply:
Are you proposing to hardcode server params all the time?

Reply:
> It seems like this struct completely coping http.Server struct, but without implemented methods and some other fields. Why don't we just use http.Server struct instead?

cgf.Server struct carries the server configuration and is part of the consolidated Options struct. The decision of team members whose code is based on Options to use cgf.Server struct is dictated by explicitness and consistency

Reply:
Well, okay then, just asking. :)